org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/groovy/SmartApp.groovy

```diff
@@ -154,51 +154,49 @@ void updateConnectionId() {
     log.debug "Updated ConnectionID to ${state.connectionId}"
 }
 
-//Endpoints functions; return device data in json format
```

Comment:
All I am doing in this is moving the new changes into the old folders.  Git views this as a bunch of edits rather than just replacing the files of the same name.  Any changes are the ones we already have in opent2t.p.* folders.

---
In reply to: [90067590](https://github.com/openT2T/translators/pull/130#discussion_r90067590) [](ancestors = 90067590)